### DISCUSSION TITLE
Export Parameter values of objects & relationships

### DISCUSSION THREAD

**Discussion by JaKehrl** [Spine-project/Spine-Toolbox#2053] [Support]
Hi community, 

is it possible to export the input parameters from objects and relationships as per attached image?
Also, would it be possible to export an overview of which parameter values are used in which scenario? 

<img width="473" alt="image" src="https://user-images.githubusercontent.com/124700754/227184965-ca5ab25e-2c7a-47dd-b4bf-b00581c49196.png">

**Comment by JaKehrl** [comment]
The problem got solved by creating "Exporter" blocks and linking them to the input database as per attached image. 

<img width="117" alt="image" src="https://user-images.githubusercontent.com/124700754/227231800-fa940a14-b183-4f3f-9156-882c47cb1c3c.png">


However, the problem I am currently still facing is that for each scenario, a respective folder subfolder is created. 
<img width="580" alt="image" src="https://user-images.githubusercontent.com/124700754/227232028-1f543f6e-6342-4dc5-85d6-f25cf36dbc5e.png">
<img width="512" alt="image" src="https://user-images.githubusercontent.com/124700754/227232136-a76f64a6-c37f-4506-8a96-b2de52ab7547.png">


Is there a way that one csv file is created that includes the values for all alternatives and not just scenario dependent?

**Comment by jkiviluo** [comment]
Yes, it's possible, but there are still some limitations. You need to map alternative to a column or a row in the exporter mappings, like here:
![image](https://user-images.githubusercontent.com/40472544/227441003-60fbb139-ea91-418c-a9c3-baecae3153d3.png)

The remaining issue is that you must not select any scenarios in the filter and then it will export all scenarios to the csv file. If you select one or more scenarios, they will be exported to separate files.
